overlays/equinix-overlay.yaml

```diff
@@ -125,20 +125,20 @@ relations:
 - - 'calico:etcd'
   - 'etcd:db'
 - - 'calico:cni'
-  - 'kubernetes-master:cni'
+  - 'kubernetes-control-plane:cni'
 - - 'calico:cni'
   - 'kubernetes-worker:cni'
-- - 'kubernetes-master:ceph-storage'
+- - 'kubernetes-control-plane:ceph-storage'
   - 'ceph-mon:admin'
```

Comment:
```suggestion
```

We dropped the `ceph-storage` relation in https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/pull/219